py_gtfs_rt_ingestion/pyproject.toml

```diff
@@ -20,7 +20,6 @@ pytest-cov = "^3.0.0"
 black = "^22.3.0"
 pylint = "^2.15.5"
 mypy = "^0.961"
-awscli = "^1.25.8"
```

Comment:
we don't use this dependency and it was causing a bad poetry lock file to be generated, so i just removed it.